doc/source/data/batch_inference.rst

```diff
@@ -73,9 +73,9 @@ For how to configure batch inference, see :ref:`the configuration guide<batch_in
 
             # Use 2 parallel actors for inference. Each actor predicts on a
             # different partition of data.
-            scale = ray.data.ActorPoolStrategy(size=2)
+            concurrency = 2
```

Comment:
With the new API (which is much better) I don't think putting this on a separate line makes a lot of sense. Maybe inline it below and say `Step 3: Map the Predictor over the dataset to get predictions. Use 2 parallel actors for inference. Each actor predicts on a different partition of data.` as the comment?

Reply:
yes, updated.